src/source/Signaling/LwsApiCalls.h

```diff
@@ -18,7 +18,6 @@ extern "C" {
 #define SIGNALING_SERVICE_TCP_KEEPALIVE_IN_SECONDS                3
 #define SIGNALING_SERVICE_TCP_KEEPALIVE_PROBE_COUNT               3
 #define SIGNALING_SERVICE_TCP_KEEPALIVE_PROBE_INTERVAL_IN_SECONDS 1
-#define SIGNALING_SERVICE_WSS_PING_PONG_INTERVAL_IN_SECONDS       10
```

Comment:
Is this ok to do? Wondering if anything would break removing this or what the alternate is in 4.2.0 of libwebsockets